pkg/virt-handler/vm.go

```diff
@@ -756,14 +756,16 @@ func (d *VirtualMachineController) defaultExecute(key string,
 	// set true to ensure that no updates to the current VirtualMachineInstance state will occur
 	forceIgnoreSync := false
 
-	log.Log.V(3).Infof("Processing vmi %v, existing: %v\n", vmi.Name, vmiExists)
+	log.Log.Infof("Processing event %v", key)
 	if vmiExists {
-		log.Log.V(3).Infof("vmi is in phase: %v\n", vmi.Status.Phase)
+		log.Log.Infof("Vmi %s in phase %v", vmi.Name, vmi.Status.Phase)
```

Comment:
how about changing that to VMI?

Reply:
not sure I understood. Do you mean Vmi -> VMI ?

Reply:
yes